okhttp-tests/src/test/java/com/squareup/okhttp/internal/http/ThreadInterruptTest.java

```diff
@@ -45,6 +59,14 @@
   // The size of the socket buffers in bytes.
   private static final int SOCKET_BUFFER_SIZE = 256 * 1024;
 
+  // 203.0.113.255 reserved for documentation.
+  private static final byte[] IPV4_BLACKHOLE_ADDRESS =
```

Comment:
To switch I think it requires that we handle the UnknownHostException exception from getByAddress(): if we just throw it the class might fail to initialize if we don't have both IPv6 and IPv4. We could set it to null in that case, I suppose.